src/main/java/io/appium/java_client/youiengine/YouiEngineDriver.java

```diff
@@ -23,6 +23,11 @@
 
 import java.net.URL;
 
+@Deprecated
```

Comment:
I suppose the common practice is to put a docsting before an annotation, but not after